packages/firestore/src/platform_node/grpc_connection.ts

```diff
@@ -113,7 +113,17 @@ export class GrpcConnection implements Connection {
         ? grpc.credentials.createSsl()
         : grpc.credentials.createInsecure();
       this.cachedStub = {
-        stub: new this.firestore.Firestore(this.databaseInfo.host, credentials),
+        stub: new this.firestore.Firestore(
+          this.databaseInfo.host,
+          credentials,
+          {
+            // gRPC will automatically perform exponential backoff if there is already a connection
+            // to this host. In production this is not an issue because "firestore.googleapis.com"
+            // resolves to many different IPs, so we almost always get a fresh connection. For the
+            // Firestore emulator, which often runs on localhost, this backoff is undesireable.
```

Comment:
Fundamentally, I think we don't want gRPC backoff ever, because we're doing our own.  So I don't think we need to mention production vs. emulator, multiple IPs, etc.  I'd simplify to something along the lines of:

```
// We do our own connection backoff (that for example is aware of whether or
// not a write stream error is permanent or not) so we don't want gRPC to do
// backoff on top of that.
```

I think I'd also be okay setting this to 0ms if that is an option.  This is essentially what we get with our webchannel_connection.ts since webchannel has no builtin backoff.  So I'm comfortable keeping gRPC in parity.

Reply:
Bleh, it's being picky about numers:
```
E1119 17:56:21.535736171   48711 channel_args.cc:345]        grpc.max_reconnect_backoff_ms ignored: it must be >= 100
```

Reply:
Yeah, there are hardcoded lower bounds for these values: https://github.com/grpc/grpc/blob/master/src/core/ext/filters/client_channel/subchannel.cc#L511

100ms is not terrible, I'll file an issue in the grpc repo to see if there's any objection to allowing clients to totally disable backoff.

Reply:
Awesome, thanks!  100ms is definitely fine for now.